recipes/gdal/post_3.5.0/conanfile.py

```diff
@@ -0,0 +1,889 @@
+            self.requires("libgta/1.2.1")
+
+        if self.options.with_hdf4:
+            self.requires("hdf4/4.2.15")
+
+        if self.options.with_hdf5:
+            self.requires("hdf5/1.13.1")
+
+        if self.options.with_heif:
+            self.requires("libheif/1.12.0")
+
+        if self.options.with_jpeg == "libjpeg":
+            self.requires("libjpeg/9d")
+        elif self.options.with_jpeg == "libjpeg-turbo":
+            self.requires("libjpeg-turbo/2.1.2")
+
+        if self.options.with_kea:
+            self.requires("kealib/1.4.14")
+
+        if self.options.with_libdeflate:
+            self.requires("libdeflate/1.12")
+
+        if self.options.with_libiconv:
+            self.requires("libiconv/1.17")
+
+        if self.options.with_libkml:
+            self.requires("libkml/1.3.0")
+
+        if self.options.with_libtiff:
+            self.requires("libtiff/4.3.0")
+
+        if self.options.with_lz4:
+            self.requires("lz4/1.9.3")
+
+        if self.options.with_mongocxx:
+            self.requires("mongo-cxx-driver/3.6.6")
+
+        if self.options.with_mysql == "libmysqlclient":
+            self.requires("libmysqlclient/8.0.29")
+        elif self.options.with_mysql == "mariadb-connector-c":
+            self.requires("mariadb-connector-c/3.1.12")
+
+        if self.options.with_netcdf:
+            self.requires("netcdf/4.8.1")
+
+        if self.options.with_odbc:
+            self.requires("odbc/2.3.9")
+
+        if self.options.with_openjpeg:
+            self.requires("openjpeg/2.5.0")
+
+        if self.options.with_openssl:
+            self.requires("openssl/1.1.1o")
+
+        if self.options.with_pcre:
+            self.requires("pcre/8.45")
+
+        if self.options.with_pcre2:
+            self.requires("pcre2/10.40")
+
+        if self.options.with_pg:
+            self.requires("libpq/14.2")
+
+        if self.options.with_png:
+            self.requires("libpng/1.6.37")
+
+        if self.options.with_podofo:
+            self.requires("podofo/0.9.7")
+
+        if self.options.with_poppler:
+            self.requires("poppler/21.07.0")
+
+        if self.options.with_proj:
+            self.requires("proj/9.0.0")
+
+        if self.options.with_qhull:
+            self.requires("qhull/8.0.1")
+
+        if self.options.with_sqlite3:
+            self.requires("sqlite3/3.38.5")
+
+        if self.options.with_webp:
+            self.requires("libwebp/1.2.2")
+
+        if self.options.with_xerces:
+            self.requires("xerces-c/3.2.3")
+
+        if self.options.with_xml2:
+            self.requires("libxml2/2.9.14")
+
+        if self.options.with_zlib:
+            self.requires("zlib/1.2.12")
+
+        if self.options.with_zstd:
+            self.requires("zstd/1.5.2")
+
+    def validate(self):
+        if self.options.get_safe("with_crypto"):
+            raise ConanInvalidConfiguration("with_crypto option has been replaced by with_openssl")
```

Comment:
no need to introduce deprecated values

```suggestion
```

Reply:
as explained in https://github.com/conan-io/conan-center-index/pull/11694#discussion_r970541914
this will make it easier for users using gdal < 3.5 with this option when they upgrade.

Reply:
removed